src/www/app/app.ts

```diff
@@ -153,61 +153,79 @@ export class App {
     let messageKey: string;
     let messageParams: string[] = [];
     let buttonKey: string;
     let buttonHandler: () => void;
     let buttonLink: string;
 
-    if (e instanceof errors.VpnPermissionNotGranted) {
+    if (error instanceof errors.VpnPermissionNotGranted) {
       messageKey = 'outline-plugin-error-vpn-permission-not-granted';
-    } else if (e instanceof errors.InvalidServerCredentials) {
+    } else if (error instanceof errors.InvalidServerCredentials) {
       messageKey = 'outline-plugin-error-invalid-server-credentials';
-    } else if (e instanceof errors.RemoteUdpForwardingDisabled) {
+    } else if (error instanceof errors.RemoteUdpForwardingDisabled) {
       messageKey = 'outline-plugin-error-udp-forwarding-not-enabled';
-    } else if (e instanceof errors.ServerUnreachable) {
+    } else if (error instanceof errors.ServerUnreachable) {
       messageKey = 'outline-plugin-error-server-unreachable';
-    } else if (e instanceof errors.FeedbackSubmissionError) {
+    } else if (error instanceof errors.FeedbackSubmissionError) {
       messageKey = 'error-feedback-submission';
-    } else if (e instanceof errors.ServerUrlInvalid) {
+    } else if (error instanceof errors.ServerUrlInvalid) {
       messageKey = 'error-invalid-access-key';
-    } else if (e instanceof errors.ServerIncompatible) {
+    } else if (error instanceof errors.ServerIncompatible) {
       messageKey = 'error-server-incompatible';
-    } else if (e instanceof OperationTimedOut) {
+    } else if (error instanceof OperationTimedOut) {
       messageKey = 'error-timeout';
-    } else if (e instanceof errors.ShadowsocksStartFailure && this.isWindows()) {
+    } else if (error instanceof errors.ShadowsocksStartFailure && this.isWindows()) {
       // Fall through to `error-unexpected` for other platforms.
       messageKey = 'outline-plugin-error-antivirus';
       buttonKey = 'fix-this';
       buttonLink = 'https://s3.amazonaws.com/outline-vpn/index.html#/en/support/antivirusBlock';
-    } else if (e instanceof errors.ConfigureSystemProxyFailure) {
+    } else if (error instanceof errors.ConfigureSystemProxyFailure) {
       messageKey = 'outline-plugin-error-routing-tables';
       buttonKey = 'feedback-page-title';
       buttonHandler = () => {
         // TODO: Drop-down has no selected item, why not?
         this.rootEl.changePage('feedback');
       };
-    } else if (e instanceof errors.NoAdminPermissions) {
+    } else if (error instanceof errors.NoAdminPermissions) {
       messageKey = 'outline-plugin-error-admin-permissions';
-    } else if (e instanceof errors.UnsupportedRoutingTable) {
+    } else if (error instanceof errors.UnsupportedRoutingTable) {
       messageKey = 'outline-plugin-error-unsupported-routing-table';
-    } else if (e instanceof errors.ServerAlreadyAdded) {
+    } else if (error instanceof errors.ServerAlreadyAdded) {
       messageKey = 'error-server-already-added';
```

Comment:
It would be cleaner if we just saved the message text here, instead of tracking the key and params:
```ts
messageText = this.localize('error-server-already-added', 'serverName', this.getServerDisplayName(error.server));
```